packages/terra-site/src/examples/status/StatusArrange.jsx

```diff
@@ -2,8 +2,9 @@
 import React from 'react';
 import Arrange from 'terra-arrange';
 import Status from 'terra-status';
+import placeholderPic150x150 from '../../assets/150x150.jpg';
 
```

Comment:
Should we update from using `<img>` to our `Image` component while we are updating the examples?